Update charter to concentrate responsibilities on the TSC rather than its voting members (#1360)
nodejs/TSC: TSC-Charter.md

## Section 7. Voting

For internal project decisions, Collaborators shall operate under Lazy
Consensus. The TSC shall establish appropriate guidelines for
implementing Lazy Consensus (e.g. expected notification and review time
periods) within the development process.

> The TSC follows a [Consensus Seeking][] decision making model.

Author: ```suggestion
The TSC follows a [Consensus Seeking][] decision making model among voting members.
```

Collaborators may opt to elevate significant or controversial
modifications, or modifications that have not found consensus to the TSC
for discussion by assigning the `tsc-agenda` tag to a pull request or

> issue. TSC votes should serve as the final arbiter where required.

Reviewer: I think this paragraph should be left as-is.

Reviewer 2: +1

Author: Would the following work better for everyone?
```suggestion
issue. The TSC should serve as the final arbiter where required.
```

Reviewer: I don't think we are in agreement with the change. The TSC voting members _are_ the final arbiter.

Reviewer 3: I think leaving it as "TSC voting members" removes ambiguity. I think characterizing it as the TSC as a whole is friendly, but not precise. I'd favor precision in this case.

Author: Maybe it's because I don't interpret English as you'all, but to me it adds some ambiguity: it could mean that I, as a TSC voting member, can serve as the final arbiter. But if we use "the TSC", it's very clear that we talk about the collective, not the individuals.
Would Gireesh suggestion (https://github.com/nodejs/TSC/pull/1360#discussion_r1145729795) work for everyone? Or does someone else has a suggestion? Or should I just shut up about it already 😅?

Reviewer: Ah, I understand where you are coming from @aduh95. While English is not my native language either, I think `the TSC` is not precise.

I think something like the following could work:

> When consensus cannot be reached, the TSC voting members should serve as the final arbiter by casting a vote.

Wdyt?

Author: I love it, and I agree your suggestion in indeed less ambiguous.
```suggestion
issue. When consensus cannot be reached, the TSC voting members should serve as the
final arbiter by casting a vote.
```

> issue. TSC votes should serve as the final arbiter where required.

Reviewer 4: ```suggestion
issue. TSC should serve as the final arbiter where required, and specifically it's voting members, when consensus cannot be reached.
```

Reviewer 5: (its)

Collaborators may opt to elevate significant or controversial
modifications, or modifications that have not found consensus to the TSC
for discussion by assigning the `tsc-agenda` tag to a pull request or

> issue. When consensus cannot be reached, the TSC voting members should serve as the
> final arbiter by casting a vote.
> The TSC will maintain and publish a list of current Collaborators, as
> well as a development process guide for Collaborators and Contributors
> looking to participate in the development effort.

Author: ```suggestion
issue. When consensus cannot be reached, the TSC voting members should serve as
the final arbiter by casting a vote. The TSC will maintain and publish a list of
current Collaborators, as well as a development process guide for Collaborators
and Contributors looking to participate in the development effort.
```
